eis_toolkit/model_performance_estimation/model_performance_estimation.py

```diff
@@ -0,0 +1,35 @@
+import sklearn
+from sklearn.model_selection import KFold, LeaveOneOut, StratifiedKFold
+
+from eis_toolkit.exceptions import InvalidCrossValidationSelected, InvalidNumberOfSplit
+
+
+def performance_model_estimation(
+    cross_validation_type: str = "LOOCV", number_of_split: int = 5
+) -> sklearn.model_selection:
+    """
+    Evaluate the feature importance of a sklearn classifier or linear model.
+
+    Parameters:
+        cross_validation_type: Select cross validation (LOOCV, SKFOLD, KFOLD).
+        number_of_split: number used to split the dataset.
```

Comment:
Could detail here what is required for the `number_of_split` so that exceptions are not raised.